joomla/40-backend-template
# [RFC] Custom Elements

Pull Request for Issue https://github.com/joomla/40-backend-template/issues/281 .

### Summary of Changes
This is a demo just to showcase that CE are easy to implement

### Testing Instructions
Apply this Pr and check the bottom part of Atum:

![screen shot 2016-12-14 at 20 21 55](https://cloud.githubusercontent.com/assets/3889375/21195110/1245d77c-c23b-11e6-83ee-afc86a242164.png)

### Documentation Changes Required
